infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -62,6 +73,38 @@
     public static boolean serverRunning = false;
 
     /**
+     * Metrics: meter (counter), histogram
+     */
+    static private final String mp = "corfu.server.";
+    static private final String mpBase = mp + "base.";
+    static private final String mpLU = mp + "logunit.";
+    static private final String mpSeq = mp + "sequencer.";
+    static private final String mpLayout = mp + "layout.";
+
+    public static final MetricRegistry metrics = new MetricRegistry();
+    static final Timer timerPing = metrics.timer(mpBase + "ping");
+    static final Timer timerVersionRequest = metrics.timer(mpBase + "version-request");
+    static final Timer timerLogWrite = metrics.timer(mpLU + "write");
+    static final Timer timerLogCommit = metrics.timer(mpLU + "commit");
+    static final Timer timerLogRead = metrics.timer(mpLU + "read");
+    static final Timer timerLogGcInterval = metrics.timer(mpLU + "gc-interval");
+    static final Timer timerLogForceGc = metrics.timer(mpLU + "force-gc");
+    static final Timer timerLogFillHole = metrics.timer(mpLU + "fill-hole");
+    static final Timer timerLogTrim = metrics.timer(mpLU + "trim");
+    static final Timer timerSeqReq = metrics.timer(mpSeq + "token-req");
+    static final Counter counterTokenSum = metrics.counter(mpSeq + "token-sum");
+    static final Counter counterToken0 = metrics.counter(mpSeq + "token-0");
+    static final Timer timerLayoutReq = metrics.timer(mpLayout + "request");
+    static final Timer timerLayoutBootstrap = metrics.timer(mpLayout + "bootstrap");
+    static final Timer timerLayoutSetEpoch = metrics.timer(mpLayout + "set-epoch");
+    static final Timer timerLayoutPrepare = metrics.timer(mpLayout + "prepare");
+    static final Timer timerLayoutPropose = metrics.timer(mpLayout + "propose");
+    static final Timer timerLayoutCommitted = metrics.timer(mpLayout + "committed");
+    static final MetricSet metricsJVMGC = new GarbageCollectorMetricSet();
+    static final MetricSet metricsJVMMem = new MemoryUsageGaugeSet();
+    static final MetricSet metricsJVMThread = new ThreadStatesGaugeSet();
```

Comment:
Based on discussion this afternoon, we settled on "none of the above" and instead a new class CorfuMetrics; did I get that right?